shop/src/main/java/ro/msg/learning/shop/entities/OrderDetails.java

```diff
@@ -0,0 +1,23 @@
+package ro.msg.learning.shop.entities;
+import lombok.Data;
+import javax.persistence.*;
+import java.io.Serializable;
+
+
+@Entity @Data
+@Table(name="order_details")
+public class OrderDetails implements Serializable {
+    @EmbeddedId OrderDetailsId orderDetailsId;
```

Comment:
This a correct way to create a compound primary key, but in was specified to use artificial primary key instead of compound ones.

> Create a data model for your application in a Flyway SQL migration. You can refine the model by adding unique indexes (e.g. supplier name) and introducing artificial PKs instead of composite ones and also add a unique index.

Reply:
I suggested to use composite primary keys. So no changes needed.